packages/transport-grpc/src/send-get-transaction-status.js

```diff
@@ -47,14 +50,14 @@ export async function sendGetTransactionStatus(ix, context = {}, opts = {}) {
   let ret = context.response()
   ret.tag = ix.tag
   ret.transactionStatus = {
-    blockId: null,
+    blockId: nonEmptyU8ToHex(res.getBlockId_asU8(), context),
     status: res.getStatus(),
     statusString: STATUS_MAP[res.getStatus()],
     statusCode: res.getStatusCode(),
     errorMessage: res.getErrorMessage(),
     events: events.map(event => ({
       type: event.getType(),
-      transactionId: u8ToHex(event.getTransactionId_asU8(), context),
+      transactionId: nonEmptyU8ToHex(event.getTransactionId_asU8(), context),
```

Comment:
Is this ever empty? AFAIK events will always have a `transactionId` associated. I think we should use the old `u8ToHex` in this case?